dist/images/ovnkube.sh

```diff
@@ -867,9 +867,9 @@ ovn-master() {
       disable_snat_multiple_gws_flag="--disable-snat-multiple-gws"
   fi
 
-  disable_pkt_mtu_check_flag=
-  if [[ ${ovn_disable_pkt_mtu_check} == "true" ]]; then
-      disable_pkt_mtu_check_flag="--disable-pkt-mtu-check"
+  disable_gateway_mtu_flag=
```

Comment:
This is an ovnkube-master configuration so need to pass it to ovnkube.sh cmd line in line 927 so this conifg is populated in master